### PR TITLE
Update fulltext.adoc

### PR DESCRIPTION
**Pull request by vstruchkov**
I think that realisation of this feature fix this issue https://github.com/neo4j-contrib/neo4j-apoc-procedures/issues/738

### CODE REVIEW

File: docs/fulltext.adoc [hunk 1]
```diff
@@ -146,7 +146,7 @@ All https://neo4j.com/docs/java-reference/3.1/#indexing-create-advanced[standard
 | name | value | description
 | type | fulltext/exact | type of the index
 | to_lower_case | false/true | if terms should be converted to lower case before indexing
-| analyzer | classname | classname of lucene analyzer to be used for this index
+| analyzers | classnames | list of classname of lucene analyzer to be used for this index
```
Comment (jexp) [inline docs/fulltext.adoc]: why the change in the config name?

File: docs/fulltext.adoc [hunk 2]
```diff
@@ -252,3 +252,29 @@ Again we're search for "Johnny", expecting now two of them:
 CALL apoc.index.search("people", "Johnny") YIELD node, weight
 RETURN node.name, weight
 ----
+
+=== Fulltext Index and multilingual string
+By default standart configuration of fultext index use standart analyzer for extract token from text.
+For support fultext search on multilangual content we need to set additional configuration to parameter `<option>` in function `apoc.index.addAllNodes`.
+
+Make sure `apoc.multilingual.enabled=true` is set.
```
Comment (jexp) [inline docs/fulltext.adoc]: where is the defined in the code?

File: docs/fulltext.adoc [hunk 3]
```diff
@@ -252,3 +252,29 @@ Again we're search for "Johnny", expecting now two of them:
...
+
+Bellow there is example of creating fulltext index with multilingual support
+[source,cypher]
+----
+CALL apoc.index.addAllNodes('locations',{
+  Person:  ["address"],
+  Address: ["address"]}, {analyzer:["English","Russian","Thai"]})
```
Comment (jexp) [inline docs/fulltext.adoc]: uses analyzer here without an `s`?

File: docs/fulltext.adoc (hunk 1 above)
Comment (vstruchkov) [reply docs/fulltext.adoc]: It's mistake. The right string is "anayzer" but you must pay attention that analizer is array now

File: docs/fulltext.adoc (hunk 2 above)
Comment (vstruchkov) [reply docs/fulltext.adoc]: I write "apoc.multilingual.enabled=true" if you decide to make multilingual index optional

File: docs/fulltext.adoc (hunk 3 above)
Comment (vstruchkov) [reply docs/fulltext.adoc]: See my comment for first questions.

Comment (vstruchkov) [reply docs/fulltext.adoc]: In this code I found error. After parameter "analizer" I had to write correct class names for analyzer

File: docs/fulltext.adoc (hunk 2 above)
Comment (jexp) [reply docs/fulltext.adoc]: But where is the code that uses it?

Comment (vstruchkov) [reply docs/fulltext.adoc]: There is no code that use it.

File: docs/fulltext.adoc (hunk 3 above)
Comment (jexp) [reply docs/fulltext.adoc]: Could you update your PR then with the fixes?

**Comment by jexp** [comment]
Hey @vstruchkov as Neo4j 3.5.x added fulltext procedures to the product we plan to deprecated and remove the ones in APOC so I guess we might not need to add this anymore. What do you think?

**Comment by vstruchkov** [comment]
Cool. 
I test it in new version
